src/core/toga/widgets/canvas.py

```diff
@@ -8,6 +8,11 @@
 from .base import Widget
 
 
+EVENODD = "evenodd"
+NONZERO = "nonzero"
+FILLRULES = [NONZERO, EVENODD]
```

Comment:
When we've got a collection of related constants, we've been moving towards using `enum` objects